spec/simplecov_helper.rb

```diff
@@ -70,6 +39,43 @@ def merge_results
       SimpleCov::ResultMerger.store_result(result)
     end
   end
+
+  def add_filters
+    add_filter 'app/controllers/concerns/accountable.rb'
+    add_filter 'config/initializers/clamscan.rb'
+    add_filter 'lib/config_helper.rb'
+    add_filter 'lib/feature_flipper.rb'
+    add_filter 'lib/gibft/configuration.rb'
+    add_filter 'lib/ihub/appointments/response.rb'
+    add_filter 'lib/salesforce/configuration.rb'
+    add_filter 'lib/vet360/address_validation/configuration.rb'
+    add_filter 'lib/search/response.rb'
+    add_filter 'lib/vet360/exceptions/builder.rb'
+    add_filter 'lib/vet360/response.rb'
+    add_filter 'modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb'
+    add_filter 'modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb'
+    add_filter 'modules/claims_api/app/swagger/*'
+    add_filter 'modules/claims_api/lib/claims_api/health_checker.rb'
+    add_filter 'lib/bip_claims/configuration.rb'
+    add_filter 'version.rb'
+  end
+
+  def add_modules
+    add_group 'Policies', 'app/policies'
```

Comment:
This change will break the way that the module generator inserts entries into the simplecov helper when a new module is created. https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/generators/module/module_generator.rb#L65-L66

Reply:
It uses the # Modules comment to insert a new entry

Reply:
If you could kindly keep the comment OR update to after: "def add_modules\n"

Reply:
Another approach would be to use a data structure, Array in the case of filters and just iterate over them.  This eliminates the complexity of the method while also stating that the list of "filters" are a data object. Something like:

```ruby

# Not All Filters Added For Brevity.
FILTERS = [
'config/initializers/clamscan.rb'
  'lib/config_helper.rb'
  'lib/feature_flipper.rb'
  'lib/gibft/configuration.rb'
  'lib/ihub/appointments/response.rb'
  'lib/salesforce/configuration.rb'
  'lib/vet360/address_validation/configuration.rb'
  'version.rb'
]

def add_filters 
  FILTERS.each { |f| add_filter f }
end
```

Something similar could be done with groups, I'd probably do use some magic though.

```ruby
GROUPS = {
  serializers: 'app/serializers',
  services:    'app/services'
}

def add_groups 
  GROUPS.each do { |k, v| add_group(k.classify, v) }
end
```

Reply:
I would defer to @LindseySaari on what all it would break though.

Reply:
Ooo that's clean, I like it @jimiray! Unfortunately, it would break the generator logic and the cleanup may be better suited for a separate ticket/PR associated with the generator.